src/version.mjs

```diff
@@ -188,12 +189,20 @@ async function resolveRelease(range) {
 
 /** @param {string} url */
 async function fetchWithRetries(url, maxRetries = 5) {
+  const dispatcher = new HttpClient().getAgentDispatcher(url);
+
   let sleepMs = 250;
   let iterationCount = 0;
   while (true) {
     iterationCount++;
     try {
-      const res = await fetch(url);
+      const res = await fetch(
+        url,
+        // dispatcher is `ProxyAgent | undefined`, which is assignable to `Dispatcher | undefined` because
+        // ProxyAgent extends Dispatcher, but TS2322 is reported.
+        // @ts-ignore:
+        { dispatcher },
```

Comment:
Without this, following ts error is shown.

```
error: TS2322 [ERROR]: Type 'ProxyAgent | undefined' is not assignable to type 'Dispatcher | undefined'.
  Property 'compose' is missing in type 'ProxyAgent' but required in type 'Dispatcher'.
```

undici's ProxyAgent extends undici's Dispatcher, so it seems false-positive.
Is this @ts-ignore acceptable?